### ISSUE TITLE
[BUG] flash limit is incorrect

### ISSUE THREAD

**Issue by shadow578** [shadow578/Marlin-H32#40]
due to a bug in the linker script, compiling marlin may create binaries that are too large to fit onto the flash of the chip.

please ensure that the memory overview shown during compilation shows a __maximum__ of `212992 B` in the 'Used Size' column for 'FLASH'. Anything larger than that may cause crashes.

```
Memory region         Used Size  Region Size  %age Used
           FLASH:      186848 B       256 KB     71.28%
             OTP:          0 GB       1020 B      0.00%
             RAM:       10800 B       188 KB      5.61%
         RET_RAM:          0 GB         4 KB      0.00%
```

to reduce the used flash, please disable marlin features that are not needed, or switch to less flash-heavy alternatives (e.g. use `AUTO_BED_LEVELING_BILINEAR` instead of `AUTO_BED_LEVELING_UBL`)

**Comment by shadow578** [comment]
note: a fix is currently beign created

**Comment by shadow578** [comment]
issue will be fixed by https://github.com/shadow578/Marlin-H32/pull/41

**Comment by shadow578** [comment]
> issue will be fixed by #41

PR merged
